generators/setup-docker/templates/fabric-docker/docker-compose.yaml

```diff
@@ -106,7 +106,9 @@ services:
       <%_ } _%>
       <%_ chaincodes.forEach(function(chaincode) { _%>
       - "$CHAINCODES_BASE_DIR/<%= chaincode.directory %>/:/var/hyperledger/cli/<%= chaincode.name %>/"
-      - "../fabric-config/collections/<%= chaincode.name %>.json:/var/hyperledger/cli/collections/<%= chaincode.name %>.json"
+      <%_ if(chaincode.privateDataConfigFile) { _%>
```

Comment:
hm... didn't knew `_%>`. what it does?

Reply:
see the docs: https://ejs.co/

`_%>` trims the following white spaces. this is probably some legacy, I would use `<% xxx -%>`